backend_tv/lifter.cpp

```diff
@@ -2842,7 +2872,25 @@ class arm2llvm {
       updateReg(added, ptrReg);
       break;
     }
+    case AArch64::LDRWroX: {
+      unsigned size;
+      if (opcode == AArch64::LDRWroX)
+        size = 4;
+      else
+        assert(false);
+      auto &op0 = CurInst->getOperand(0);
```

Comment:
this whole first part isn't needed -- you get the operands and do the assertions over again in the helper function

Reply:
sorry I mean lines 2881-2887 should be removed